### PR TITLE
WIP: Writing instructions for making changes to group website

### PR DESCRIPTION
**Pull request by MagicMilly**
Included so far:

- launching website locally from a forked repo
- staying up-to-date with changes on the original repo
- adding to the People section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## How to Make Changes to the Group Website
+
+### A. From your own forked repo
+
+#### I. Getting Started
+
+1. Fork the original [repo](https://github.com/az-digitalag/group-website)
```
Comment (dlebauer) [inline README.md]: * If in the org, option to just create a branch?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+2. If you do not already have RStudio installed, follow these [instructions](https://rstudio.com/products/rstudio/download/)
+3. Open RStudio and navigate to your forked repo
+4. If not already downloaded, install the blogdown and hugo packages using these [instructions](https://bookdown.org/yihui/blogdown/installation.html)
+5. To preview the website locally, use the command `blogdown::serve_site()` in the console
+6. You should now be able to preview the group website within RStudio, or copy and paste the url provided in the console into your browser. 
+
+#### II. Keeping your forked repo up to date with new changes from the command line
```
Comment (dlebauer) [inline README.md]: Can this just reference / be a link to our group wiki in the section on our standard / generic git workflow?

File: README.md (hunk 1 above)
Comment (MagicMilly) [reply README.md]: Yes I was going to include that as well

File: README.md (hunk 2 above)
Comment (MagicMilly) [reply README.md]: Good idea!

**Comment by KristinaRiemer** [comment]
We should add a section about how to create a new blog post, including how to add images. Briefly, I've been saving image files in `content/post`, but then putting them into the blog post using the addin Insert Image. This creates a copy (I think?) of the image in `static/post/[blog post name]`. I think we might not need to have the images in the first location at all? So clear instructions about how to add images to blog posts correctly are in order.

File: README.md (hunk 1 above)
Comment (KristinaRiemer) [reply README.md]: I think most of these instructions should be the same whether it's from a branch off the main repo, a fork of the main repo, or a branch off a fork. So I'll make some suggestions for how to handle that.

Comment (KristinaRiemer) [reply README.md]:
```suggestion
1. Either clone the original [repo](https://github.com/az-digitalag/group-website) to your local computer and always work off of a non-master branch, or fork the original repo and clone the forked repo to your computer. You can additionally work off of a branch on the forked repo if you'll be working on multiple different tasks at a time. 
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+## How to Make Changes to the Group Website
+Note: General instructions for git / git workflow can be found on our group's OSF [page](https://osf.io/tzmhp/wiki/Using%20Git%20and%20GitHub/)
+
+### A. From your own forked repo
+
+#### I. Getting Started
+
+1. Fork the original [repo](https://github.com/az-digitalag/group-website)
+2. If you do not already have RStudio installed, follow these [instructions](https://rstudio.com/products/rstudio/download/)
+3. Open RStudio and navigate to your forked repo
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion
3. Open RStudio and create a new project for your local cloned website repo
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+#### III. How to add a blog post
+
+1. Within the repo in RStudio, open the `content` > `post` folders
+2. As with the People section, copy an existing blog post and rename it using the `YYYY-MM-DD-name-of-post.md` format
+3. Change the metadata at the top of the file as needed (i.e. `title`, `authors`, `date`, and `slug`)
+4. Use Markdown [syntax](https://www.markdownguide.org/basic-syntax/) to write the post
+5. To add images to your blog post, use the `Add-ins` drop down menu within RStudio and select `Insert Image`
+5. Commit and push your local changes to your forked repo and create a pull request in the original repo with these changes. 
+
+### B. From a branch in az-digitalag/group-website repo
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion

```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+## How to Make Changes to the Group Website
+Note: General instructions for git / git workflow can be found on our group's OSF [page](https://osf.io/tzmhp/wiki/Using%20Git%20and%20GitHub/)
+
+### A. From your own forked repo
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion

```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+#### I. Getting Started
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion
#### I. Getting started
```

File: README.md (hunk 4 above)
Comment (KristinaRiemer) [reply README.md]:
```suggestion
#### IV. Contributing changes back to website

1. Once you're happy with your changes, add and commit the changed files. 
2. Push commit(s) from local repo back to the main repo and open a pull request. 
3. Find someone to review, make comments, and merge pull request. 
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+1. Fork the original [repo](https://github.com/az-digitalag/group-website)
+2. If you do not already have RStudio installed, follow these [instructions](https://rstudio.com/products/rstudio/download/)
+3. Open RStudio and navigate to your forked repo
+4. If not already downloaded, install the blogdown and hugo packages using these [instructions](https://bookdown.org/yihui/blogdown/installation.html)
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion
4. If not already downloaded, install the blogdown R package and Hugo using these [instructions](https://bookdown.org/yihui/blogdown/installation.html)
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+5. To preview the website locally, use the command `blogdown::serve_site()` in the console
+6. You should now be able to preview the group website within RStudio, or copy and paste the url provided in the console into your browser. 
+
+#### II. How to add a team member to the People section
+
+1. Within the repo in RStudio, open the `content` > `people` folders
+2. Create a folder with the naming format of `new-person`
+3. To include a photo or headshot in the bio, upload a `.png` image to that person's folder named `featured.png`
+4. Copy `index.md` from another team member's folder and update with new information. Remember to change the team member tag to `current`, `affiliate`, or `past` as needed.
+5. Commit and push your local changes to your forked repo and create a pull request in the original repo with these changes.
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion

```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+5. Commit and push your local changes to your forked repo and create a pull request in the original repo with these changes. 
+
+#### III. How to add a blog post
+
+1. Within the repo in RStudio, open the `content` > `post` folders
+2. As with the People section, copy an existing blog post and rename it using the `YYYY-MM-DD-name-of-post.md` format
+3. Change the metadata at the top of the file as needed (i.e. `title`, `authors`, `date`, and `slug`)
+4. Use Markdown [syntax](https://www.markdownguide.org/basic-syntax/) to write the post
+5. To add images to your blog post, use the `Add-ins` drop down menu within RStudio and select `Insert Image`
+5. Commit and push your local changes to your forked repo and create a pull request in the original repo with these changes.
```
Comment (KristinaRiemer) [inline README.md]:
```suggestion

```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+4. Copy `index.md` from another team member's folder and update with new information. Remember to change the team member tag to `current`, `affiliate`, or `past` as needed.
+5. Commit and push your local changes to your forked repo and create a pull request in the original repo with these changes. 
+
+#### III. How to add a blog post
+
+1. Within the repo in RStudio, open the `content` > `post` folders
+2. As with the People section, copy an existing blog post and rename it using the `YYYY-MM-DD-name-of-post.md` format
+3. Change the metadata at the top of the file as needed (i.e. `title`, `authors`, `date`, and `slug`)
+4. Use Markdown [syntax](https://www.markdownguide.org/basic-syntax/) to write the post
+5. To add images to your blog post, use the `Add-ins` drop down menu within RStudio and select `Insert Image`
```
Comment (KristinaRiemer) [inline README.md:27]:
```suggestion
5. To add images to your blog post, use the `Add-ins` drop down menu within RStudio and select `Insert Image`. Once you've added the image to the post, feel free to delete the copy of the image from your local file system
```

Comment (KristinaRiemer) [inline README.md:27]: I'm not sure if my suggestion is clear? I mean to delete the image file from the Downloads or wherever it came from, not from the repo itself. Maybe this isn't even necessary to include?

**Review by KristinaRiemer — APPROVED** [review]
I made some suggested edits, mostly related to git stuff and a few other small edits. Feel free to take or leave!

File: README.md (hunk 10 above)
Comment (MagicMilly) [reply README.md:27]: Hmm I don't think I quite understand. If you mean the instruction to delete the download from your local machine, I agree that it's probably not necessary to include.

**Pull request merged**